### DISCUSSION TITLE
HTTP project unable to load resources over HTTPS code-server proxy

### DISCUSSION THREAD

**Discussion by PixNyb** [coder/code-server#5093] [Q&A] [answered]
Hello all,
I’ve been getting into code-server because i want to develop my web applications on my iPad, however my local projects usually do not have an ssl certificate nor do they have a web server advanced enough to handle those. And so all requests made are over http. Usually this wouldn’t be an issue. But because my code-server is secured none of these projects can load their resources. Is there any way to force all outgoing traffic over https?

**Comment by jsjoeio** [comment, marked as answer]
Hmm 🤔 Is code-server running behind HTTPS? If so you could try using the [built-in proxy](https://coder.com/docs/code-server/latest/guide#using-a-subpath) or if you need an absolute path, try [this](https://coder.com/docs/code-server/latest/guide#using-a-subpath).

And if that doesn't work, you could probably use `ngrok` or `localtunnel` to expose those over HTTPS and access in another tab on your iPad.

**Reply by PixNyb** [reply]
Thank you for your reply! The built in proxy solutions didn't work for me as they forced ssl. However localtunnel does not and allows me to access my local projects over http instead of https. Everything works as intended now.
